src/chart/custom/CustomSeries.ts

```diff
@@ -338,7 +338,7 @@ export interface CustomSeriesOption extends
     /**
      * @deprecated
      */
-    itemStyle?: ItemStyleOption;
+    itemStyle?: ItemStyleOption<TCbParams>;
```

Comment:
Could we use `CallbackDataParams` here as other series did?

Reply:
Thank you for the suggestion! Fixed.